apps/condo/domains/resident/tasks/helpers/sendBillingReceiptNotifications.js

```diff
@@ -26,8 +26,8 @@ const sendBillingReceiptNotifications = async (context = null) => {
 
         return { status: DISABLED }
     }
-
-    await sendResidentsNoAccountNotifications()
+    //TODO: DOMA-10913 This func needs to be refactored and optimized or removed, currently it falls by time out
+    // await sendResidentsNoAccountNotifications()
```

Comment:
This function fails for organizations with huge amount of BillingProperties. But still works for small organizations.